seagap/configs/main.py

```diff
@@ -110,9 +112,35 @@ def customise_sources(
             )
 
 
+class OutputPath(BaseModel):
+    """Output path base model."""
+
+    path: str = Field(
+        ...,
+        description="Path string to the sound speed data. Ex. s3://bucket/ctd_sound_speed.dat",
+    )
+    storage_options: Dict[str, Any] = Field(
```

Comment:
Are there specific keys/values that are supported? Could this be strongly typed instead?

Reply:
This can't be specified as it changes based on which cloud service remote protocol used. i.e. AWS S3 (s3), Google Cloud Storage (gcs), Azure Data Lake (adl).